content/en/cd-as-a-service/reference/deployfile/ref-deployment-file.md

```diff
@@ -206,9 +205,10 @@ targets:
       beforeDeployment:
```

Comment:
Unrelated to this PR: we are missing the spec for `afterDeployment` constraints

Reply:
Agree. The entire ref-deployment-file should be double-checked. I know I haven't been keeping it updated based on new fields. Big ooops.